features/embedded-wallets/turnkey-embedded-wallet.ts

```diff
@@ -0,0 +1,52 @@
+import { onPasskeyCreate } from "@/utils/passkeys/create-passkey";
+import { IEmbeddedWallet } from "./embedded-wallet.interface";
+import { Client } from "@xmtp/react-native-sdk";
```

Comment:
Would be nice to have `libs/xmtp/index.ts`  or `libs/xmtp/client.ts` and do `export const XmtpClient = Client` . So we do `XmtpClient.` instead